frontend/src/components/home.jsx

```diff
@@ -1,14 +1,16 @@
 import React, { Component } from "react";
 import CarouselComponent from './carousel'
-
+import UpperMid from './Midsection/UpperMid'
+import LowerMid from './Midsection/LowerMid'
 import "./home.css";
 
 class Home extends Component {
   render() {
     return (
-      <div className="Home container">
-        <h1>YMIM</h1>
+      <div className="Home">
         <CarouselComponent />
+        <UpperMid />
```

Comment:
I'm confused about the logic of separating these sections. What was the thought process behind that?